src/backend/columnar/sql/citus_columnar--11.1-1--11.2-1.sql

```diff
@@ -1 +1,61 @@
 -- citus_columnar--11.1-1--11.2-1
+
+CREATE OR REPLACE FUNCTION columnar_internal.columnar_ensure_am_depends_catalog()
```

Comment:
Doing this here because columnar-Makefile doesn't know how to expand `#include` directives. At least comment this in the downgrade script ?